decodeWays.py

```diff
@@ -0,0 +1,29 @@
+class Solution(object):
+    def numDecodings(self, s):
+        """
+        :type s: str
+        :rtype: int
+        """
+        if s == '' or s[0] == '0':
+            return 0
+        if len(s) == 1:
+            return 1
+        prePreCount, preCount, curCount = 1, 1, 0
+        for i in range(1, len(s)):
+            flag1 = self.decodeTwoDigit(s[i - 1: i + 1])
+            flag2 = self.decodeDigit(s[i])
+            if flag1 == 0 and flag2 == 0 :
```

Comment:
`flag1` and `flag2` shall be returned as boolean values...
Maybe should do something like
```
if not flag1 and not flag2
```